### PR TITLE
Use bg = None for floating window

### PR DESCRIPTION
**Pull request by dchinmay2**

### CODE REVIEW

File: teal/notifier/status.tl [hunk 1]
```diff
@@ -54,6 +54,9 @@ function StatusModule._create_win()
       col = vim.o.columns,
       zindex = 10,
     })
+    local winhlns = api.nvim_create_namespace("notifier_winhl")
```
Comment (vigoux) [inline teal/notifier/status.tl]: Why creating a new name space? We aldreay have one, I think we should use it.

File: teal/notifier/status.tl [hunk 2]
```diff
@@ -54,6 +54,9 @@ function StatusModule._create_win()
...
+    api.nvim_win_set_hl_ns(StatusModule.win_nr, winhlns)
+    api.nvim_set_hl(winhlns, "NormalFloat", { bg = "None" })
```
Comment (vigoux) [inline teal/notifier/status.tl:59]: We should expose a new highlight group here.

**Review by vigoux — CHANGES_REQUESTED** [review]

File: teal/notifier/status.tl (hunk 2 above)
Comment (dchinmay2) [reply teal/notifier/status.tl:59]: Are your sure? This is just for the background of the message which I think everyone would want to be same as `Normal` background

File: teal/notifier/status.tl (hunk 1 above)
Comment (dchinmay2) [reply teal/notifier/status.tl]: Will do

File: teal/notifier/status.tl (hunk 2 above)
Comment (vigoux) [reply teal/notifier/status.tl:59]: Having it configurable but with a sane default is pretty much equivalent.
I'd like to have it configurable.

Comment (dchinmay2) [reply teal/notifier/status.tl:59]: I can't think of a good way to do this, making a highlight group called `NotifierFloat` with `bg = "None"` and linking the floatwin's `NormalFloat` to `NotifierFloat` doesn't work

Comment (vigoux) [reply teal/notifier/status.tl:59]: I'll look into it then.
But it should work just like any other highlight group.
Maybe `bg=None` should be added to other highlight groups too.

Comment (dchinmay2) [reply teal/notifier/status.tl:59]: It is different from other highlight groups - putting `bg = None` in other `NotifierXX` groups (or any other group on a floatwin) will use `NormalFloat`'s bg, just like putting `bg=None` in a normal (non-floating) window's highlight group will use `Normal`'s bg

Comment (vigoux) [reply teal/notifier/status.tl:59]: 'kay, then I'll check that.
Do you mind if I fix the issue on my end ?

**Comment by vigoux** [comment]
This feature should be implemented in latest master, could you confirm it is the case ?

**Comment by dchinmay2** [comment]
Yes, thanks!

**Pull request closed without merging**